js/views/modals/Settings/Advanced/Settings.js

```diff
@@ -0,0 +1,108 @@
+import app from '../../../../app';
+import loadTemplate from '../../../../utils/loadTemplate';
+import baseVw from '../../../baseVw';
+import Appearance from './Appearance';
+import Transactions from './Transactions';
+import Server from './Server';
+import SMTPIntegration from './SMTPIntegration';
+import AppearanceSettings from '../../../../models/AppearanceSettings';
+import TransactionSettings from '../../../../models/TransactionSettings';
+import ServerSettings from '../../../../models/ServerSettings';
+import SMTPIntegrationSettings from '../../../../models/SMTPIntegrationSettings';
+
+export default class Advanced extends baseVw {
+  constructor(options = {}) {
+    super({
+      className: 'settingsAdvanced',
+      ...options
+    });
+
+    this.options = options;
+    this.settings = app.settings.clone( );
+
+
+    self.app = app;
```

Comment:
@rmisio debug instrumentation. So I could check what was going on with app settings in console.